src/main/java/org/assertj/core/api/AbstractCharSequenceAssert.java

```diff
@@ -182,6 +182,28 @@ public SELF isNotBlank() {
     return myself;
   }
 
+/**
+   * Verifies that the actual {@code CharSequence} hace mixed case:
+   * These assertions succeed:
+   * <pre><code class='java'> assertThat("Apple").isMixedCase();
+   * assertThat("Mango").isMixedCase();
+   * assertThat("Cherry").isMidexxCase();</code></pre>
+   *
+   * Whereas these assertions fail:
+   * <pre><code class='java'> assertThat("himanshu ").isMixedCase();
+   * assertThat("gavy").isMixedCase();
+   * assertThat("ALPHA").isMixedCase();
+   * String nullString = null;
+   * assertThat(nullString)isMixedCase();</code></pre>
+   *
+   * @return {@code this} assertion object.
```

Comment:
Could you please align the javadoc to the style of `isUpperCase` or `isLowerCase`?

Reply:
Done